### PR TITLE
Parse batch.csv rows from header instead of index

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -202,29 +202,34 @@ def write_csv_for_mturk(path, all_assignments, approved_assignments, rejected_as
 assignments_to_approve = Set.new
 assignments_to_reject = {}
 
+columns = {}
+
 is_header = true
 CSV.foreach('batch.csv') do |row|
-  # Skip header row
```
Comment (paked) [inline app.rb:207]: Why did you remove this comment?

File: app.rb [hunk 2]
```diff
@@ -202,29 +202,34 @@ def write_csv_for_mturk(path, all_assignments, approved_assignments, rejected_as
...
-  assignment_id = row[14]
+  assignment_id = row[columns['AssignmentId']]
 
   assignments[assignment_id] = {
-    hit_id: row[0],
-    worker_id: row[15],
-    school: row[27],
-    region: row[28],
-    found_teacher: row[30],
+    hit_id: row[columns['HITId']],
```
Comment (paked) [inline app.rb:221]: How would you feel about abstracting these `columns[*]` keys?

The lookup could become: `hit_id: row[COLUMN_HITId]`, with the constant `COLUMN_HITId = columns['HITId']` up the top of this file.

**Review by paked — COMMENTED** [review]
A few quick questions and suggestions. Almost a LGTM 😄

File: app.rb (hunk 1 above)
Comment (maxwofford) [reply app.rb:207]: We aren't skipping the header row anymore. Now we save the index of each column of the header. https://github.com/hackclub/process-mturk-results/pull/1/files#diff-cc95738088603531796e0d0f246a5d77R211

File: app.rb (hunk 2 above)
Comment (maxwofford) [reply app.rb:221]: Would the constant at the top of the file be able to handle a different number/order of columns in the CSV?

**Comment by maxwofford** [comment]
@paked Just replied to your comments

**Comment by zachlatta** [comment]
Closing this because we're no longer using the repo.

**Comment by zachlatta** [comment]
Sorry :-(.
